bba/bba.go

```diff
@@ -244,6 +253,12 @@ func (bba *BBA) handleBvalRequest(sender cleisthenes.Member, bval *BvalRequest)
 }
 
 func (bba *BBA) handleAuxRequest(sender cleisthenes.Member, aux *AuxRequest) error {
+	// check delayed message and wait until bval is set
+	if !bba.bvalSet() {
+		bba.incomingAuxReqRepo.Save(bba.round.value(), sender.Address, aux)
```

Comment:
Is this really needed logic? checking `!bba.bvalSet()`?

Reply:
maybe legacy.. i'll check